src/pages/Admin/Room/Edit/RoomEditModal.tsx

```diff
@@ -183,6 +183,24 @@ export const RoomEditModal: React.FC<RoomEditModalProps> = ({
             </div>
           </Form.Row>
 
+          <Form.Row>
+            <div className="room-edit-modal__input">
+              <Form.Label>Change room type</Form.Label>
+              <Form.Control
+                as="select"
+                custom
+                name="type"
+                ref={register}
+                defaultValue={room.type}
+              >
+                <option />
+                {Object.values(RoomTypes).map((roomType) => (
```

Comment:
This might be a useful thing to extract as a little helper in types/rooms. I can imagine we may want to do similar in other places in the app in future.

```typescript
export const allRoomTypes: string[] = () => Object.values(RoomTypes)
```

Reply:
I wonder, do we want to change the way the label of these is shown so it looks 'nicer' to a human viewing it? Either all lowercase, or even better maybe: Titlecase

Reply:
Not going with this change for now as React Bootstrap select doesn't use the value property in the traditional sense and uses keys and state to manage the values. It was resulting in option label being set as option value.

Reply:
@TroySK You 👍🏻 'd my suggestion in https://github.com/sparkletown/sparkle/pull/1305/files#r609141461 yet don't seem to have made any code changes for it?

Reply:
Will be doing the change once I am able to implement https://github.com/sparkletown/sparkle/pull/1305#discussion_r617310471 correctly as I plan to use the method there.